lib/cnb/version.rb

```diff
@@ -1,3 +1,3 @@
 module CNB
-  VERSION = '2.0.0'
+  VERSION = '2.0.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.